project/profiles/capacity4more/modules/c4m/content/c4m_content_group/plugins/content_types/c4m_content_group.inc

```diff
@@ -48,3 +48,15 @@ function c4m_content_group_details() {
 
   return drupal_render($node_view);
 }
+
+/**
+ * Implements hook_preprocess().
+ */
+function c4m_content_group_preprocess_node(&$variables) {
+  if ($variables['view_mode'] != 'group_dashboard') {
+    return;
+  }
+
+  $variables['theme_hook_suggestions'] = ['node__' . $variables['node']->type . '__group_dashboard'];
```

Comment:
As I understand it's adding it. When I dpm `variables` I seeing an array named `theme_hook_suggestions` with all the templates name.

By the way, this is the way it's done in other parts of the project.